src/main/java/Task/Add.java

```diff
@@ -0,0 +1,22 @@
+package task;
+
+/**
+ * The `Add` class represents an "add" task in the BloopBot application.
+ * It extends the base `Task` class and inherits its properties and methods.
+ * An `Add` task includes a description and can be marked as completed or not completed.
+ * This class is used to represent tasks created using the "add" command.
+ *
+ * @author raydenlim
+ * @version 0.0.0
+ */
+public class Add extends Task {
```

Comment:
Is this a necessary class? Add is a verb and perhaps be more suited as a method rather than a separate class